### ISSUE TITLE
Vue 3 Object(...) is not a function

### ISSUE THREAD

**Issue by ReazerDev** [probil/vue-socket.io-extended#528]
Hi, I'm using alpha5 and when I'm trying to use the Socket Decorator I get a TypeError in the function beneath it:
`Object(...) is not a function`.
![image](https://user-images.githubusercontent.com/36013882/117345786-3cef0180-aea7-11eb-8071-de6685dab3d2.png)
When I remove the Decorator, the TypeError is gone.

Also when running npm run serve, I get a warning:
![image](https://user-images.githubusercontent.com/36013882/117345835-4bd5b400-aea7-11eb-8ce3-38355294c30a.png)

**Comment by jnt0r** [comment]
Are you sure you're using alpha5? Newest I can found is alpha4. And there everything is working for me using @Socket Decorator.
Maybe you could provide some more context of your code. The first thing I would think of is the import of Socket. Maybe it's not correct.

**Comment by jnt0r** [comment]
Okay, I need to correct my comment. Got the warning on compile-time too. Remembered that there were some breaking changes in alpha stated here https://github.com/probil/vue-socket.io-extended/issues/489#issuecomment-771497973

**Comment by probil** [comment]
@ReazerDev In v5 decorator is supposed to be imported from the sub-path:
```js
import { Socket } from 'vue-socket.io-extended' // v4
import Socket from 'vue-socket.io-extended/decorator' // v5
```
could you check whether it works this way?

**Comment by ReazerDev** [comment]
> Are you sure you're using alpha5? Newest I can found is alpha4. And there everything is working for me using @socket Decorator.
> Maybe you could provide some more context of your code. The first thing I would think of is the import of Socket. Maybe it's not correct.

Nope I've been using alpha-4. I'm updating to alpha-5 now.

**Comment by ReazerDev** [comment]
![image](https://user-images.githubusercontent.com/36013882/118323579-20834280-b501-11eb-8834-5d6018d34855.png)
Now I get this error when compiling.

This is how I import Socket:
`import Socket from 'vue-socket.io-extended/decorator';`

VSCode doesn't throw any errors and I can even Ctrl+Right-Click it and it finds the type definitions.
I've deleted my node_modules folder and my package-lock.json.

This is in my package.json:
`"vue-socket.io-extended": "^5.0.0-alpha.5"`

**Comment by probil** [comment]
@ReazerDev That's weird. How are use transpiling TS in vue files? With typescript or babel? Or maybe something more cutting edge, e.g. esbuild?

**Comment by ReazerDev** [comment]
I'm using babel

**Comment by jnt0r** [comment]
I just updated to alpha.5 and it's working for me. Think I'm using babel too. 
@ReazerDev can you share a link to your repo or provide some snippets of your config?

**Comment by ReazerDev** [comment]
I'll push the repo to Github and post a link here, once I'm home👍

**Comment by ReazerDev** [comment]
https://github.com/ReazerDev/vue-3-socket-io-demo

**Comment by jnt0r** [comment]
@ReazerDev got your repo working. Just updated the dependencies to the newest ones. Especially the ones with "@vue/...".

**Comment by ReazerDev** [comment]
Wait I think I just now realised, that you mean, that everything is working and not that you got my repo working and are starting to look what causes my issue?

**Comment by jnt0r** [comment]
No. I meant that I got your repo working by updating the dependencies. You need to update the dependencies with "@vue/..." to get your repo working.
